src/rvoice/fluid_rvoice_mixer.c

```diff
@@ -164,15 +166,22 @@ fluid_rvoice_mixer_process_fx(fluid_rvoice_mixer_t *mixer, int current_blockcoun
         for(f = 0; f < mixer->fx_units; f++)
         {
             int buf_idx = f * fx_channels_per_unit + SYNTH_REVERB_CHANNEL;
-            
-            for(i = 0; i < current_blockcount * FLUID_BUFSIZE; i += FLUID_BUFSIZE)
+            int samp_idx = buf_idx * FLUID_MIXER_MAX_BUFFERS_DEFAULT * FLUID_BUFSIZE;
+            int sample_count = current_blockcount * FLUID_BUFSIZE;
+
+            /* in mix mode, map fx out_rev at index f to a dry buffer at index dry_idx */
+            if(mixer->mix_fx_to_out)
+            {
+                /* dry buffer mapping, should be done more flexible in the future */
```

Comment:
Why do you think that "dry buffer mapping, should be done more flexible in the future"? IMO, if one wants a more flexible way, one should use `fluid_synth_process()`, in which case `mixer->mix_fx_to_out` will be false. And if we wanted to bring this "more flexible" mapping to the audio drivers as well (e.g. via an extra setting or so), we could/should/would use `fluid_synth_process()`.

Reply:
This comment isn't sufficient and confusing. Please see comment below about  "dry buffer mapping, should be done more flexible in the future".